src/features/likes/likes-slice.jsx

```diff
@@ -6,13 +6,14 @@ const initialState = {
   status: null,
 };
 
-let encodedToken = "abcnvdjnv3r34rndnv";
+let encodedToken = "";
 
 export const addToLikes = createAsyncThunk(
   "likes/addToLikes",
   async (video) => {
```

Comment:
Inside this function, there is second parameter called "thunkAPI" so with the help of that you can access store. You can implement it. Instead of taking token from localestorage